pkg/api/v1alpha1/cluster_types.go

```diff
@@ -458,7 +471,8 @@ type ClusterGenerate struct {
 	metav1.TypeMeta `json:",inline"`
 	ObjectMeta      `json:"metadata,omitempty"`
 
-	Spec ClusterSpec `json:"spec,omitempty"`
+	Spec   ClusterSpec   `json:"spec,omitempty"`
+	Status ClusterStatus `json:"status,omitempty"`
```

Comment:
Interesting, curious why you are adding this here
I would vote to not include it in the generate struct since this should never be changed manually by a human (neither in disk with the cli or through gitops)

Reply:
Ah yes. Thanks for catching this. I meant to remove this when removing the other generate logic for this